hammers/scripts/enforce_retirement.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+
+import os
+import sys
+from hammers import MySqlArgs, query
+from hammers.slack import Slackbot
+from hammers.util import base_parser
+
+def correct_state(db,slk,dryrun=False):
+
+    # Find retired nodes
+    retired_nodes = query.find_reservable_retired_nodes(db)
+    node_list = []
+    for node in retired_nodes:
+        node_list.append(node['uuid'])
+        if not dryrun:
+            blazar_fix = query.blazar_set_non_reservable(db, node['uuid'])
+            assert blazar_fix == 1
+    
+    if not dryrun:
+        mess = ("Reverted state of node(s) " + str(', '.join(node_list))  + " to non-reservable.")
+        db.db.commit()
```

Comment:
Do you need this commit statement? I think the above block should have already committed the changes.

Reply:
I don't think so, the nodes weren't updated in the db until I added the parentheses at the end (that's what I was just asking you about :) )